event_listener/dfusion_db/event_pusher.py

```diff
@@ -41,8 +41,8 @@ def update_accounts(event: Dict[str, Union[int, str, str, int]]):
     :return: bson.objectid.ObjectId
     """
     transition_type = event['transitionType']
-    state_index = event['stateIndex']
-    state_hash = event['stateHash']
+    state_index = event['from']
```

Comment:
this was changed intentionally... are you sure we still want to and from? Are you working with old contracts?

Reply:
was working with old contracts